src/rust-std.mk

```diff
@@ -0,0 +1,49 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG                     := rust-std
+$(PKG)_WEBSITE          := https://www.rust-lang.org/
+$(PKG)_VERSION          := 1.37.0
+
+$(PKG)_DEPS             := cc $(BUILD)~cargo $(BUILD)~rustc $(BUILD)~rust-std-bootstrap get-target-triplet
+
+ifneq (, $(findstring darwin,$(BUILD)))
+    BUILD_TRIPLET = $(firstword $(call split,-,$(BUILD)))-apple-darwin
+else
+    ifneq (, $(findstring linux,$(BUILD)))
+        BUILD_TRIPLET = $(firstword $(call split,-,$(BUILD)))-unknown-linux-gnu
+    else
+        BUILD_TRIPLET = $(BUILD)
+    endif
+endif
+
+define DEP_TARGET
+$(shell [[ -f $(MXE_TMP)/get-target-triplet.txt ]] && cat $(MXE_TMP)/get-target-triplet.txt || echo ERROR)
```

Comment:
If using double-brackets is an issue you can quote the argument like so:
```
[ -f "$(MXE_TMP)/get-target-triplet.txt" ]
```

Reply:
Actually yeah, I'm not entirely sure why I used the double brackets instead of the alternate test format.